records_mover/records/job/mover.py

```diff
@@ -35,6 +35,6 @@ def run_records_mover_job(source_method_name: str,
         source = source_method(**source_kwargs)
         target = target_method(**target_kwargs)
         return records.move(source, target, processing_instructions)
-    except Exception as e:
-        logger.error(e)
+    except Exception:
+        logger.error('', exc_info=True)
```

Comment:
This line wasn't logging anything, but the exception raised to the top of the stack did.  With these two changes, we now send the exception through the filtered logging stream only:

```
$ mvrec table2file redshift vbroz test foo.csv
19:26:43 - Starting...
19:26:49 - Mover: copying from TableRecordsSource(redshift) to DataUrlTarget by filling in a temporary location...
19:26:49 - Starting Redshift unload...
19:26:49 - 
Traceback (most recent call last):
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.SyntaxError: syntax error at or near "FROM2"
LINE 2:        UNLOAD ('SELECT * FROM2 vbroz.test') TO 's3://vince-s...
                ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/broz/src/records-mover/records_mover/records/job/mover.py", line 37, in run_records_mover_job
    return records.move(source, target, processing_instructions)
  File "/Users/broz/src/records-mover/records_mover/records/mover.py", line 120, in move
    return records_target.move_from_temp_loc_after_filling_it(records_source,
  File "/Users/broz/src/records-mover/records_mover/records/targets/data_url.py", line 74, in move_from_temp_loc_after_filling_it
    records_source.\
  File "/Users/broz/src/records-mover/records_mover/records/sources/table.py", line 93, in move_to_records_directory
    export_count = self.driver.unload(schema=self.schema_name, table=self.table_name,
  File "/Users/broz/src/records-mover/records_mover/db/redshift/redshift_db_driver.py", line 109, in unload
    return self._redshift_unloader.unload(schema=schema,
  File "/Users/broz/src/records-mover/records_mover/db/redshift/unloader.py", line 93, in unload
    out = self.unload_to_s3_directory(schema, table, unload_plan, s3_directory)
  File "/Users/broz/src/records-mover/records_mover/db/redshift/unloader.py", line 72, in unload_to_s3_directory
    self.db.execute(unload)
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 2191, in execute
    return connection.execute(statement, *multiparams, **params)
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 984, in execute
    return meth(self, multiparams, params)
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/sql/elements.py", line 293, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1097, in _execute_clauseelement
    ret = self._execute_context(
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1287, in _execute_context
    self._handle_dbapi_exception(
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1481, in _handle_dbapi_exception
    util.raise_(
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/Users/broz/.pyenv/versions/3.8.1/envs/records-mover-3.8.1/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.SyntaxError) syntax error at or near "FROM2"
LINE 2:        UNLOAD ('SELECT * FROM2 vbroz.test') TO 's3://vince-s...
                ^

[SQL: 
       UNLOAD (%(select)s) TO %(unload_location)s
       CREDENTIALS %(credentials)s
       MANIFEST
       
       
       DELIMITER AS %(delimiter)s
       
       
       
       
       
       ESCAPE
       
       
       
       
    ]
[parameters: {'select': 'SELECT * FROM2 vbroz.test', 'unload_location': 's3://vince-scratch/a-GZ8hYUa2g/', 'credentials': 'aws_access_key_id=AKIAJ5RGGUDLXXMDAYHQ;aws_secret_access_key=****************************************', 'delimiter': ','}]
(Background on this error at: http://sqlalche.me/e/f405)$ 
```